docs/integrations/dell.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "Dell"
+url: dell
+---
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+
+# Iceberg Dell Integration
+
+## Dell ECS Integration
+
+Iceberg can be used with Dell's Enterprise Object Storage (ECS) by using the ECS catalog since 0.14.0.
+
+Dell ECS has many features that make Iceberg a highly compatible table format, such as append operations for file writers and content addressable storage (CAS) for table commits. See [Dell ECS](https://www.dell.com/en-us/dt/storage/ecs/index.htm) for more information on Dell ECS.
+
+### Connection parameters
+
+When using Dell ECS with Iceberg, these configuration parameters are required:
+
+| Name                     | Description             |
+| ------------------------ | ----------------------- |
+| ecs.s3.endpoint          | ECS S3 service endpoint |
+| ecs.s3.access-key-id     | ECS Username            |
+| ecs.s3.secret-access-key | S3 Secret Key           |
+
+An ECS catalog requires that you configure a warehouse location where all data and metadata will be created.
+
+| Example                    | Description                                                     |
+| -------------------------- | --------------------------------------------------------------- |
+| ecs://bucket-a             | Use the whole bucket as the data                                |
+| ecs://bucket-a/namespace-a | Use a prefix to access the data only in this specific namespace |
+
+When you provide the `warehouse`, the last `/` will be ignored. The `ecs://bucket-a` is same with `ecs://bucket-a/`.
+
+### Runtime dependencies
+
+The Iceberg `runtime` jar supports different versions of Spark and Flink. If the version was not matched in the example, please check the related document of Spark and Flink.
+
+Even though the [Dell ECS client](https://github.com/EMCECS/ecs-object-client-java) jar is backward compatible, Dell EMC still recommends using the latest version of the client.
+
+### Spark
+
+For example, to use the Dell ECS catalog with Spark 3.2.1, you should create a Spark session like:
+
+```python
```

Comment:
Let's just use java/scalar or bash example for keep consistentence as other pages.

Reply:
Now the scripts are bash-script.